docs/blog/connected-registry.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Connected Registry Private Preview
+description: Private preview for ACR connected registry feature.
+ms.topic: post
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
```

Comment:
@toddysm nit: "We are announcing the private preview of the Azure Container Registry (ACR) connected registry feature.". Added a "the" before Azure Container Registry. Alternately